Exception when storing a new article work (not in a collection)
The article was stored but an exception was received. The corresponding file was not stored. F, [2019-02-22T16:42:27.598812 #9785] FATAL -- : [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] ActiveRecord::RecordNotFound (Co uldn't find Sipity::Workflow with [WHERE sipity_workflows.active = ? AND sipity_workflows.permission_template_id IN (SELECT permission_templates.id FROM permission_templates WHERE permission_templates.source_id IS NULL)]): F, [2019-02-22T16:42:27.598848 #9785] FATAL -- : [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] F, [2019-02-22T16:42:27.598909 #9785] FATAL -- : [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] activerecord (5.1.6.1) lib/activ e_record/relation/finder_methods.rb:340:in raise_record_not_found_exception!' [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] activerecord (5.1.6.1) lib/active_record/relation/finder_methods.rb:129:in first !' [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] hyrax (2.4.1) app/models/sipity/workflow.rb:34:in find_active_workflow_for' [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] hyrax (2.4.1) app/models/concerns/hyrax/in_admin_set.rb:10:in active_workflow' [8da9406b-19af-4f9b-9f01-6bb5aab3f6c5] hyrax (2.4.1) app/services/hyrax/workflow/workflow_factory.rb:51:in `create_workf low_entity!'